src/index.ts

```diff
@@ -209,6 +209,10 @@ const plugin: JupyterFrontEndPlugin<void> = {
           await handleNotebookSharing(notebookPanel, sharingService, true);
         } catch (error) {
           console.error('Error in share command:', error);
+        } finally {
+          if (tracker.currentWidget) {
```

Comment:
You would probably want to save the `currentWidget` on a variable before `try` and reuse it, otherwise it can be a different notebook you will delete.